benchmark/httpapi.py

```diff
@@ -163,6 +188,55 @@ def delete(self, request, id):
         request.setResponseCode(NO_CONTENT)
         return self.backend.delete(id)
 
+    @app.route("/benchmark-results", methods=['GET'])
+    def query(self, request):
+        """
+        Query the previously stored benchmarking results.
+
+        Currently this method supports filtering only by the branch name.
+        There is no support for the results paging, but a limit on the number
+        of the results is supported.
+        The order of the results is fixed at the moment and it's by
+        by the result timestamp in the descending order.
```

Comment:
Remove the duplicate "by" and change "in the descending order" to just "in descending order".